python/ray/train/v2/api/exceptions.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, controller_failure: Exception):
             "Training failed due to controller error:\n" + str(controller_failure)
         )
         self.controller_failure = controller_failure
+        self.with_traceback(controller_failure.__traceback__)
```

Comment:
can we do this for worker failures as well and remove the UserExceptionWithTraceback? I didn't realize you could do this at the time

Reply:
error = WorkerGroupError(worker_failures). # Just pass the base exceptions with_traceback instead of UserExceptionWithTraceback

str(error) == summarized error traces for each of the ranks (aka [this logic](https://github.com/ray-project/ray/blob/78082d65fa7081172d2848ced56d68cc612f8fd1/python/ray/train/v2/_internal/execution/worker_group/poll.py#L66))